src/Bot/Dubtrack.hs

```diff
@@ -63,11 +69,39 @@ songLink (songType -> SongTypeSoundcloud) =
   "Soundcloud links are not supported yet"
 songLink _ = error "This should never happen Kappa"
 
+newtype RoomName = RoomName
+  { unName :: T.Text
+  }
+
+instance IsEntity RoomName where
+  nameOfEntity _ = "RoomName"
```

Comment:
This will literally create an entity called "RoomName" in the database. And without any kind of context it's unclear what room we are talking about. Let's name this entity something like `DubtrackRoom`.

Reply:
Yes I agree, it's renamed

Reply:
It would be great to rename the record type from `RoomName` to `DubtrackRoom` too, so it's easier later to find the entity from database in code.

Having that said, we do have some entities that have different names in code and the db, but it's done due to old db schema compatibility. In this case there is no valid reason to introduce such discrepancy.

Reply:
Also done